### ISSUE TITLE
Fix lint issues or disable specific lint checks

### ISSUE THREAD

**Issue by krt16s** [queler/holokenmod#24]
Building the app fails with:

````
:app:lintVitalRelease/home/krt/src/fdroiddata/build/com.holokenmod/app/src/main/res/layout-small/activity_main.xml:61: Error: Duplicate id @+id/gridcontainer, already defined earlier in this layout [DuplicateIds]
        android:id="@+id/gridcontainer"
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /home/krt/src/fdroiddata/build/com.holokenmod/app/src/main/res/layout-small/activity_main.xml:31: @+id/gridcontainer originally defined here

   Explanation for issues of type "DuplicateIds":
   Within a layout, id's should be unique since otherwise findViewById() can
   return an unexpected view.

1 errors, 0 warnings
````

While I can set `android { lintOptions { abortOnError false } }` on  my side, it should actually handled by upstream, since you should know best if errors/warning might affect the app at all.

**Comment by queler** [comment]
yeah i just migrated to gradle and hadn't run a release build, so I'm looking into it

**Comment by queler** [comment]
I fixed the most severe lint errors in v1.4.1.  Gradle builds it fine from my wrapper.  I can't figure out why fdroid fails with a NPE in :app:lintVitalRelease but i think the fix you did in the metadata file will work.  I don't know why one fails and one doesn't

**Comment by krt16s** [comment]
The workaround is no longer necessary with the recent changes, thanks for the quick response!
